tox.c

```diff
@@ -351,8 +351,12 @@ static _Bool load_save(Tox *tox)
                         /* Well, lets try the current directory... */
                         data = file_raw("tox_save", &size);
                         if(!data) {
-                            /* F***it I give up! */
-                            return 0;
+                            debug("trying android hack...");
+                            data = file_raw("/data/data/tox.utox/files/tox_save.tox", &size);
```

Comment:
It's not anymore, I left that in because I was tired last night.

It was only in there while I was trying to get data_path to return
correctly.

You can remove it, or I can if you'd like?

On Sat, Mar 7, 2015 at 1:21 PM, irungentoo <notifications@github.com> wrote:

> In tox.c <https://github.com/notsecure/uTox/pull/887#discussion_r25999714>
> :
>
> > @@ -351,8 +351,12 @@ static _Bool load_save(Tox *tox)
> >                          /* Well, lets try the current directory... */
> >                          data = file_raw("tox_save", &size);
> >                          if(!data) {
> > -                            /* F***it I give up! */
> > -                            return 0;
> > +                            debug("trying android hack...");
> > +                            data = file_raw("/data/data/tox.utox/files/tox_save.tox", &size);
>
> why is this necessary?
>
> Isn't that the path datapath() returns?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/notsecure/uTox/pull/887/files#r25999714>.
>